CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updates to change-tracked properties no longer occur on the next RunLoop, fixing modal dismissal on macOS (#1450)
 - `+` characters are properly encoded as `%2B` in form events (#1449)
 - Fixed retain cycle in `LiveViewCoordinator` (#1455)
+- Made `StylesheetCache` thread-safe, fixing occasional crashes (#1461)
```

Comment:
I'm not sure we need to note this in the changelog, since the `StylesheetCache` is not part of any release yet.

Reply:
if the stylesheet cache was crashing the app due to a thread safety issue I'm OK acknowledging that here

Reply:
We have no note that `StylesheetCache` was added, so we should probably add that to the changelog too then.

Reply:
I think the cache was added before the CHANGELOG was added though, or am I mistaken?